andela/ah-frontend-tesseract
# #160814740 users should have landing page

#### What does this PR do?
implements landing page for authors haven
#### Description of Task to be completed?
- adds landing page with slider showcasing what authors haven is about.
- previews some articles from authors a given user follows
- previews random articles if the user does not follow any user as yet

#### How should this be manually tested?
- start the app using `npm start` and navigate to home 
- a landing page should be shown with sample article

#### What are the relevant pivotal tracker stories?
[#160814740](https://www.pivotaltracker.com/story/show/160814740)
#### Screenshots (if appropriate)
![screencapture-localhost-3000-2018-10-23-12_17_53](https://user-images.githubusercontent.com/30045448/47350101-bf36c200-d6bd-11e8-869a-28d8135671bb.png)
